packages/vc-status-list/src/impl/OAuthStatusList.ts

```diff
@@ -0,0 +1,165 @@
+import { IAgentContext, ICredentialPlugin } from '@veramo/core'
+import { ProofFormat, StatusListType } from '@sphereon/ssi-types'
+import {
+  CheckStatusIndexArgs,
+  CreateStatusListArgs,
+  SignedStatusListData,
+  StatusListResult,
+  StatusOAuth,
+  UpdateStatusListFromEncodedListArgs,
+  UpdateStatusListIndexArgs,
+} from '../types'
+import { determineProofFormat, getAssertedValue, getAssertedValues } from '../utils'
+import { IStatusList } from './IStatusList'
+import { StatusList, StatusListJWTHeaderParameters } from '@sd-jwt/jwt-status-list'
+import { IJwtService } from '@sphereon/ssi-sdk-ext.jwt-service'
+import { IIdentifierResolution } from '@sphereon/ssi-sdk-ext.identifier-resolution'
+import { createSignedJwt, decodeStatusListJWT } from './encoding/jwt'
+import { createSignedCbor, decodeStatusListCWT } from './encoding/cbor'
+
+type IRequiredContext = IAgentContext<ICredentialPlugin & IJwtService & IIdentifierResolution>
+
+export const BITS_PER_STATUS_DEFAULT = 2 // 2 bits are sufficient for 0x00 - "VALID"  0x01 - "INVALID" & 0x02 - "SUSPENDED"
+export const DEFAULT_LIST_LENGTH = 250000
+export const DEFAULT_PROOF_FORMAT = 'jwt' as ProofFormat
+export const STATUS_LIST_JWT_HEADER: StatusListJWTHeaderParameters = {
+  alg: 'EdDSA',
```

Comment:
Why is this fixed? If this is a default object, make it clear in the name

Reply:
removed
That came from the sd-jwt library test, should not have been there anymore